mono/culture/Makefile.am

```diff
@@ -0,0 +1,13 @@
+MAKEFLAGS := $(MAKEFLAGS) --no-builtin-rules
```

Comment:
You don't need this Makefile at all, do you?
Cannot Automake let you "reach around" and compile files in any directory from another directory?

Reply:
Two excuses, take your pick:
1. I don't know how to make `make dist` work for that.  maybe it does?
2. bare directories without a Makefile in a project that otherwise use recursive Make look suspicious.

Reply:
Ok but the real reason (also to do with `make dist`) is that in dotnet/runtime we'd like for these files _not to exist_.  And if I do conditional sources in `metadata/Makefile.am`, `make dist`  will still expect them to exist even if `ENABLE_NETCORE` is defined.  So in the end it's actuall easier to conditionally depend on a libtool convenience library